draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,61 @@ IDNA2003 {{RFC3490}}.
 
 # Privacy Considerations
 
-Cookies are often criticized for letting servers track users. For example, a
-number of "web analytics" companies use cookies to recognize when a user returns
-to a web site or visits another web site. Although cookies are not the only
-mechanism servers can use to track users across HTTP requests, cookies
-facilitate tracking because they are persistent across user agent sessions and
-can be shared between hosts.
+The primary privacy risk associated with cookies is their ability to correlate user activity. This
+can happen on a single site, but is most problematic when activity is tracked across different,
+seemingly unconnected Web sites to build a profile of the user.
+
+Over time, this capability (warned against explicitly in {{?RFC2109}} and all of its successors)
+has become widely used for varied reasons including:
+
+* authenticating users across sites,
+* assembling information on users,
+* protecting against fraud and other forms of undesirable traffic,
+* targeting advertisements at specific users or at users with specified attributes,
+* measuring how often ads are shown to users, and
+* recognizing when an ad resulted in a change in user behavior.
+
+While not every use of cookies is necessarily problematic for privacy, their potential for abuse
+has become a widespread concern in industry and society. In response to these concerns, user agents
+have actively constrained cookie functionality in various ways (as allowed and encouraged by
+previous specifications), while avoiding disruption to features they judge desirable for the health
+of the Web.
+
+It is too early to declare consensus on which specific mechanism(s) should be used to mitigate the privacy impact of cookies; ongoing developments in user agents are best characterised as experiments that
```

Comment:
I think some browsers are well beyond what would be characterized as experiments.